jcstress-wrapper/src/main/kotlin/komem/litmus/JCStressRunner.kt

```diff
@@ -0,0 +1,47 @@
+package komem.litmus
+
+import java.nio.file.Path
+import kotlin.system.exitProcess
+
+/**
+ * Note that this 'runner' is severely different from all others.
+ */
```

Comment:
Can we try to unify the interfaces and get rid of this distinction?

Reply:
I tried again with your other suggestions in mind and it generally worked out, with one problem noted as TODO (see for yourself)

Reply:
I meant the "optimize for many tests" line. Currently the interface invokes the runner one time for each test, which is suboptimal in case with jcstress. This is not a huge problem though